src/DwitTech.AccountService.WebApi/appsettings.Local.json

```diff
@@ -12,5 +12,12 @@
   "DB_USERNAME": "db-user",
   "DB_PASSWORD": "db-pass",
   "DB_NAME": "account-db",
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "Jwt": {
+    "Key": "DhdhU23hdhdn95fsksksk8kfjvoofn0vooo",
```

Comment:
key in the code!!! Yes i know it is a dummy, gitignore this or use usersecrets

Reply:
I gitignored it, but it still appeared. I'll try to figure out why.